### PR TITLE
Change DateTime format

### PR DESCRIPTION
**Pull request by IvkinStanislav**
Fix issues [440](https://github.com/svenstaro/miniserve/issues/440) and [748](https://github.com/svenstaro/miniserve/issues/748)

### CODE REVIEW

File: src/renderer.rs [hunk 1]
```diff
@@ -622,13 +620,10 @@ fn page_header(title: &str, file_upload: bool, favicon_route: &str, css_route: &
...
-    src_time.map(DateTime::<Utc>::from).map(|date_time| {
-        (
-            date_time.format("%b %e").to_string(),
-            date_time.format("%R").to_string(),
-        )
-    })
+fn convert_to_utc(src_time: Option<SystemTime>) -> Option<String> {
+    src_time
+        .map(DateTime::<Local>::from)
+        .map(|date_time| date_time.format("%Y/%m/%d %H:%M:%S %:z").to_string())
```
Comment (svenstaro) [inline src/renderer.rs:626]: Let's have these be displayed in the more traditional dash separated format:
```suggestion
        .map(|date_time| date_time.format("%Y-%m-%d %H:%M:%S %:z").to_string())
```
Alternatively, you could also use `to_rfc2822()` which might be more readable to some people. Your call.

**Review by svenstaro — CHANGES_REQUESTED** [review]
Thanks, this is good stuff. I'd just like the format to be a adjusted slightly.

**Comment by IvkinStanislav** [comment]
> Thanks, this is good stuff. I'd just like the format to be a adjusted slightly.

Ready

**Comment by svenstaro** [comment]
Thanks!

**Pull request merged**